pandas/io/parquet.py

```diff
@@ -302,6 +303,10 @@ def to_parquet(
     if isinstance(partition_cols, str):
         partition_cols = [partition_cols]
     impl = get_engine(engine)
+
+    if path is None:
+        path = io.BytesIO()
+
     return impl.write(
```

Comment:
you also need to return `path.getvalue()` no? (of path was None in the first place).

Reply:
yes! fixed that